README.md

```diff
@@ -12,6 +12,7 @@ developments that cannot be integrated into core TensorFlow
 ## Contents
 | Sub-Package    | Addon  | Reference                                  |
 |:----------------------- |:----------- |:---------------------------- |
+| addons.image | DenseImageWarp |                                      |
```

Comment:
Is it correct to use camel-case here?

Reply:
Oh, I think it's a mistake. Use real endpoint `dense_image_warp` instead.